### DISCUSSION TITLE
torrentfs

### DISCUSSION THREAD

**Discussion by anacrolix** [distribyted/distribyted#270] [General]
Hello, cool project. I'm curious why you didn't use anacrolix/torrent/fs for the fuse part of the project?

Based on your thoughts I'm considering retiring my implementation of torrentfs, maintaining my fuse bindings is problematic.

**Comment by ajnavarro** [comment]
If I remember correctly it was mainly for Windows support and to be able to stack [io/fs.FS](https://pkg.go.dev/io/fs) implementations (torrent FS -> zip FS / rar FS / 7zip FS -> FUSE / HTTP / WebDAV ).

It had been a PITA to make and keep it working on all platforms. In the future for other apps I might consider implementing WebDAV or similar and allowing the user to mount it as a network drive.

**Reply by anacrolix** [reply]
Thanks! I think I might ditch my implementation.
